src/phoenix/server/api/types/Project.py

```diff
@@ -194,6 +196,8 @@ async def spans(
             stmt = span_filter(stmt)
         if sort:
             stmt = sort.update_orm_expr(stmt)
```

Comment:
```suggestion
        if sort:
            stmt = sort.update_orm_expr(stmt)
       else:
            stmt = stmt.sort_by(desc(models.Span.start_time))
```

Reply:
addressed in https://github.com/Arize-ai/phoenix/pull/3053/commits/0c8f8a7affadef922bcdb2258374d0f7c4f1fc0b